.github/workflows/deploy-changed-users.yaml

```diff
@@ -43,9 +43,7 @@ jobs:
           if [ ! -z "$ERROR" ]
           then
             echo
-            echo Make sure you launch this job using the biomage devops tools, and not from GitHub.
-            echo Launching from GitHub directly is not supported to ensure cluster credentials are not
-            echo kept in the repository.
+            echo Make sure you the necessary secrets are defined in the respository.
```

Comment:
@aerlaut can you correct this comment -- it doesn't make sense right now

Reply:
Resolved with better comment